app/components/goals-list/goals-list.jsx

```diff
@@ -1,13 +1,28 @@
 import React from 'react';
 
+import GoalTile from '../goal-tile.jsx';
+
 const goalsList = ({goals}) => {
 
   const goalsListItems = goals.map(goal => {
-    return <li key={ goal.id }>{ goal.name }</li>
+    return <li
+      key={ goal.id }
+    >
+      <GoalTile
+        goal={ goal }
+      />
+    </li>
   });
 
+  const dynamicStyle = {
+    height: Math.max(window.innerHeight - 90, goals.length * 108 + 115),
+  };
+
   return (
-    <div className="page goalsList">
+    <div
+      className="page goalsList"
```

Comment:
yeah i get your point. I think it looks better on one line, so I've tidied this up. Only there are some cases where there are a lot of props and then it makes it far more readable to have the things on different lines. Can I not do a mixture in some cases?